requests.go

```diff
@@ -22,6 +22,9 @@ type ReqLogin struct {
 	Token                    string     `json:"token,omitempty"`
 	DeviceID                 string     `json:"device_id,omitempty"`
 	InitialDeviceDisplayName string     `json:"initial_device_display_name,omitempty"`
+	InhibitLogin             bool       `json:"inhibit_login"`
```

Comment:
`inhibit_device`, right?

Reply:
I figured both are indicative, but on a second thought, `inhibit_device` might help differentiate between `register` `inhibit_login` and this `inhibit_device`.

Reply:
Done

Reply:
@PiotrKozimor 
Done